lib/Views/src/layouts/DateTimeView64x64.cpp

```diff
@@ -286,6 +296,74 @@ void DateTimeView64x64::drawAnalogClockHand(YAGfx& gfx, int16_t minute, int16_t
         col);
 }
 
+
+void DateTimeView64x64::getConfiguration(JsonObject& jsonCfg) const
+{
+    JsonObject jsonAnalogClock = jsonCfg.createNestedObject("analogClock");
+
+    jsonAnalogClock["secondsMode"] = m_secondsMode;
+
+    for (uint32_t index = 0U;  index < ANA_CLK_COL_MAX; ++index)
+    {
+        jsonAnalogClock[ANALOG_CLOCK_COLOR_KEYS[index]]= Util::colorToHtml(m_analogColors[index]);
+    }
+}
+
+bool DateTimeView64x64::setConfiguration(const JsonObjectConst& jsonCfg)
+{
+    bool result = true;
+    JsonObjectConst jsonAnalogClock = jsonCfg["analogClock"];
+
+    if (false == jsonAnalogClock.isNull())
+    {
+        JsonVariantConst jsonSecondsMode = jsonAnalogClock["secondsMode"];
+
+        if ((false == jsonSecondsMode.is<uint8_t>()) &&
+            (SECONDS_DISP_MAX <= jsonSecondsMode.as<uint8_t>()))
+        {
+            LOG_WARNING("JSON seconds mode not found or invalid type.");
+            result = false;
+        } 
+        else
+        {
+            m_secondsMode = static_cast<SecondsDisplayMode>(jsonSecondsMode.as<uint8_t>());
+
+            for (uint32_t idx = 0U; idx < ANA_CLK_COL_MAX; ++ idx)
+            {
+                JsonVariantConst color = jsonAnalogClock[ANALOG_CLOCK_COLOR_KEYS[idx]];
+
+                if (false == color.is<String>())
+                {
+                    LOG_WARNING(
+                        "JSON attribute %s not found or invalid type.", 
+                        ANALOG_CLOCK_COLOR_KEYS[idx]);
+                    result = false;
+                }
+                else
+                {
+                    m_analogColors[idx] = Util::colorFromHtml(color);
+                }
+            }
+        }
+    }
+    
+    return result;
+}
+
+bool DateTimeView64x64::mergeConfiguration(JsonObject& jsonMerged, const JsonObjectConst& jsonSource)
+{
+    bool            result          = false;
+    JsonObjectConst jsonAnalogClock = jsonSource["analogClock"];
+
+    if (false == jsonAnalogClock.isNull())
+    {
+        jsonMerged["analogClock"] = jsonAnalogClock;

```

Comment:
Did you check the REST API if you change a color value?

Reply:
Didn't consider REST impact so far, only JSON from web page. Next page has fix for it. It works with this now:

    curl --request POST --url "http://192.168.1.112/rest/api/v1/display/uid/64860/dateTime?analogClock.secondsMode=3&analogClock.handMinCol=%2300000FF"